drivers/ukvirtio/blk/Config.uk

```diff
@@ -0,0 +1,9 @@
+config VIRTIO_BLK
+	bool "Virtio Block Device"
+	default y if LIBUKBLKDEV
+	default n
+	depends on LIBUKBLKDEV
```

Comment:
This should instead be `select LIBUKBLKDEV`
Also add `select VIRTIO`, I will provide more insight on that in a follow up comment on `virtio_bus`.